tests/test_operation.py

```diff
@@ -1058,254 +1058,25 @@ def test_arithmetic_errors(self):
 class TestDecomposition:
     """Test for operation decomposition"""
 
-    def test_decomposition_with_int_wire(self):
-        """Test that the decomposition of a non-parametrized Operation works
-        when we simply pass an integer as a wire without further specification."""
-        decomp_1 = qml.PauliRot.decomposition(0.1, "X", 1)
-        decomp_2 = qml.PauliRot.decomposition(0.1, "X", [1])
-        expected_decomp = qml.PauliRot.decomposition(0.1, "X", wires=1)
```

Comment:
were these tests and below moved elsewhere?

Reply:
All operation-specific decomposition tests were moved to the operation-specific locations. This way `test_operation.py` will test the parent abstract base class, and not the thousand and one children that inherit from it.

Reply:
Makes a lot of sense!